README.md

```diff
@@ -1,2 +1,44 @@
-# Building-with-Backgrounds-and-Gradients
-replicating an old apple page
+# Project Name
+
+> Building with Backgrounds and Gradients.
```

Comment:
[OPTIONAL]
This could be slightly more descriptive (not too many words of course) but you can specify what you mean by building with gradients and backgrounds. Like, specify the use of CSS, etc.